source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -0,0 +1,185 @@
+===========================
+Update Arrays in a Document
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to update array elements in a document.
+
+To update elements in an array, perform the following actions:
+
+- Provide an :ref:`update document <update-document>` specifying the update *(not discussed in this guide)*
+- Specify which array elements to update
+- Perform the update using an update operation with these specifications
+
+Sample Data
+~~~~~~~~~~~
+
+To run the examples in this guide, load the sample data into the
+``tea`` collection of the ``quantity`` database with the following
+snippet:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/updateArray.go
+   :language: go
+   :dedent:
+   :start-after: begin insert docs
+   :end-before: end insert docs
+
+.. include:: /includes/fundamentals/automatic-db-coll-creation.rst
+
+The following examples use the ``FindOneAndUpdate()`` function to
+retrieve and update a document, and to return the state of the document
```

Comment:
you don't need the comma here. The second clause is a dependent clause. See https://www.grammarly.com/blog/comma-before-and/.
```suggestion
retrieve and update a document and to return the state of the document
```